### ISSUE TITLE
asterisks before args and kwargs are ignored in python

### ISSUE THREAD

**Issue by zhangzq** [doxygen/doxygen#6494]
If I wrote a python function:

```py
## test
# @param *args doc
# @param **kwargs doc
def test(*args, **kwargs):
    pass
```

The `test` function will be showed as `def test(args, kwargs)`  in the function list，where the asterisks before `args` and `kwargs` are missed.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #6496

**Comment by albert-github** [comment]
Code has been integrated in master on github.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.15.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
